package.json

```diff
@@ -32,6 +33,38 @@
       "examples/with-mobx/**"
     ]
   },
+  "bundlesize": [
+    {
+      "path": "./test/integration/size-limit/.next/static/*/pages/_app.js",
+      "maxSize": "3 kB"
+    },
+    {
+      "path": "./test/integration/size-limit/.next/static/*/pages/_error.js",
+      "maxSize": "4.35 kB"
+    },
+    {
+      "path": "./test/integration/size-limit/.next/static/*/pages/index.js",
+      "maxSize": "1.46 kB"
+    },
+    {
+      "path": "./test/integration/size-limit/.next/static/*/pages/about.js",
+      "maxSize": "237 B"
+    },
+    {
+      "path": "./test/integration/size-limit/.next/static/chunks/*.js",
+    "jest-junit": "^5.0.0",
+    "jest-junit": "^5.0.0",
```

Comment:
These lines probably don’t help 😄

Reply:
agreed, my git gui screwed it up I think 🤔